prov/opx/src/fi_opx_reliability.c

```diff
@@ -1880,26 +1879,34 @@ uint64_t fi_opx_reliability_send_ping(struct fid_ep *ep,
 
 	// Send one ping to cover the entire replay range.
 	ssize_t rc = fi_opx_hfi1_tx_reliability_inject(ep,
-					key.value, dlid, rx,
+					key_value, dlid, rx,
 					psn_start,
 					psn_count,
 					FI_OPX_HFI_UD_OPCODE_RELIABILITY_PING);
 
-	INC_PING_STAT_COND(rc == FI_SUCCESS, PINGS_SENT, key.value, psn_start, psn_count);
+	INC_PING_STAT_COND(rc == FI_SUCCESS, PINGS_SENT, key_value, psn_start, psn_count);
 
 	OPX_TRACER_TRACE_RELI(OPX_TRACER_END_SUCCESS, "RELI_SEND_PING");
-	return (rc == FI_SUCCESS) ? 0 : key.value;
+
+	if(rc){
+		return OPX_RELIABILITY_PING_NO_CREDITS;
+	}
```

Comment:
I would remove the parentheses when the body has a single line.

Reply:
Per our prior discussion on this, the team would prefer to bracket every condition body, even if it's only a single line, due to the risk of introducing bugs if it's not bracketed (esp. when using macros).

I know this doesn't strictly adhere to the standard. If this is a sticking point, is it something to discuss in a meeting?

Reply:
Ok, I'll let you leave it as it is.